srcpkgs/muffin/template

```diff
@@ -16,14 +16,18 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="http://developer.linuxmint.com/projects/cinnamon-projects.html/"
 distfiles="https://github.com/linuxmint/${pkgname}/archive/${version}.tar.gz"
-checksum=e634209f1a1dfec7efc7c741e65014903ea937926070dba3793cac595df3d5a3
+checksum=a4f1cb9ed80b7ea1e51d30b97ea3ae2f196b06fc9fd56c0283e64c9ee1322c18
 shlib_provides="libmuffin-clutter-0.so libmuffin-cogl-pango-0.so
  libmuffin-cogl-path-0.so libmuffin-cogl-0.so"
 
 pre_configure() {
 	NOCONFIGURE=1 ./autogen.sh
 }
 
+pre_build() {
+	sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
```

Comment:
`vsed`

Why is it necessary?

Reply:
It is not. It is an optimisation, originated in gnome and used in the archlinux PKGBUILD. Ι can remove it.

Reply:
I think it's ok to leave it, just leave a comment on top.

Reply:
I removed it. Thanks for the review.